static/src/javascripts/bootstraps/commercial.js

```diff
@@ -79,25 +83,46 @@ define([
         secondaryModules.unshift(['cm-stickyTopBanner', stickyTopBanner.init]);
     }
 
-
     return {
         init: function () {
             if (!config.switches.commercial) {
                 return;
             }
 
             userTiming.mark('commercial start');
+            ophanTracking.addBaseline(ophanTracking.primaryBaseline);
+
+            var primaryModulePromises = [];
+
+            primaryModules.forEach(function (pair) {
+
+                var moduleName = pair[0];
 
-            var modulePromises = [];
+                robust.catchErrorsAndLog(moduleName, function () {
+                    var modulePromise = pair[1]().then(function(){
+                        ophanTracking.moduleCheckpoint(moduleName, ophanTracking.primaryBaseline);
+                    });
 
-            modules.forEach(function (pair) {
-                robust.catchErrorsAndLog(pair[0], function () {
-                    modulePromises.push(pair[1]());
+                    primaryModulePromises.push(modulePromise);
                 });
             });
 
-            Promise.all(modulePromises).then(function () {
-                robust.catchErrorsAndLogAll(secondaryModules);
+            Promise.all(primaryModulePromises).then(function () {
+                ophanTracking.addBaseline(ophanTracking.secondaryBaseline);
```

Comment:
Given that the only change in this block to the block beginning at line `93` is the `ophanTracking` object, could it be wrapped into a single function that is called with a given `ophanTracking` object?